release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -133,6 +133,15 @@ Packet data is not sent to the network flows collector. Packet-level metadata su
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc#tracking-network-flows[Tracking network flows].
 
+[id="ocp-4-8-preparation-HAProxy-2.2"]
+==== Converting HTTP header names to support upgrading to {product-title} 4.8
+
+If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+
+{product-title} updated to HAProxy 2.2, which lowercases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. Make sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading {product-title} now that HAProxy 2.2 is available.
```

Comment:
Wondering if L10N might question `lowercases` as a verb - maybe `changes HTTP header names to lowercase` instead?